wrappers/python/demo_reader.py

```diff
@@ -1,10 +1,20 @@
-import sys, zxingcpp
-from PIL import Image
+import sys
+import zxingcpp
+import cv2
+#from PIL import Image
 
-img = Image.open(sys.argv[1])
+if len(sys.argv) == 1:
+	img = cv2.imread('test.png')
+#	img = Image.open('test.png')
+else:
+	img = cv2.imread(sys.argv[1])
+#	img = Image.open(sys.argv[1])
```

Comment:
Same here, please revert all changes in this file above the `print(` line below.